index.js

```diff
@@ -64,22 +65,26 @@ module.exports = function (h, opts) {
             } else {
               key = concat(key, parts[i][2])
             }
+            isBoolean = false
           } else break
         }
         for (; i < parts.length; i++) {
           if (parts[i][0] === ATTR_VALUE) {
             if (!cur[1][key]) cur[1][key] = strfn(parts[i][1])
             else cur[1][key] = concat(cur[1][key], parts[i][1])
+            isBoolean = false
           } else if (parts[i][0] === VAR && parts[i][1] === ATTR_VALUE) {
             if (!cur[1][key]) cur[1][key] = strfn(parts[i][2])
             else cur[1][key] = concat(cur[1][key], parts[i][2])
+            isBoolean = false
           } else {
+            if (isBoolean) {
+              cur[1][key] = 'true'
+            }
             i--
             break
           }
         }
-      } else if (s === ATTR_KEY) {
```

Comment:
I believe this is dead code, so I removed it. This was probably the source of the bug.